src/Surfnet/ServiceProviderDashboard/Application/Command/PrivacyQuestions/PrivacyQuestionsCommand.php

```diff
@@ -68,6 +69,18 @@ class PrivacyQuestionsCommand implements Command
      */
     private $otherInfo;
 
+    private DpaType $dpaType;
```

Comment:
No assertion is needed here?

Reply:
A required Assertion should be set here. Thanks for spotting that!